config/locales/en.yml

```diff
@@ -219,6 +224,11 @@ en:
     all_teams: *all_teams
     no_deletion_warning: 'Team can not be deleted because its a part of some Projects'
     no_team_to_show: 'No Team to show...'
+    name_field: 'Name'
+    users_field: 'Users'
+    created_field: 'Created'
+    actions_field: 'Actions'
+    ago: 'ago'
```

Comment:
since these are common ones and can be used for multiple resources, move these to shared